miden-crypto/src/dsa/mod.rs

```diff
@@ -1,3 +1,4 @@
 //! Digital signature schemes supported by default in the Miden VM.
 
+/// Digital signature schemes supported by default in the Miden VM.
```

Comment:
This is the same as the top comment, maybe change it to something specific to RpoFalcon512